src/main/java/core/basesyntax/MyMap.java

```diff
@@ -1,7 +1,7 @@
 package core.basesyntax;
 
 public interface MyMap<K, V> {
-    void put(K key, V value);
+    V put(K key, V value);
```

Comment:
we dont need to change interface methods signature

Reply:
if I don't change the interface signature, the tests don't work

Reply:
Are you sure, it works for me

Reply:
Are you sure? It works for me